x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/host_details.tsx

```diff
@@ -74,7 +78,24 @@ const HostDetails = memo(({ details }: { details: HostMetadata }) => {
         title: i18n.translate('xpack.endpoint.host.details.policyStatus', {
           defaultMessage: 'Policy Status',
         }),
-        description: <EuiHealth color="success">active</EuiHealth>,
+        description: (
+          <EuiHealth color="success">
+            {/* eslint-disable-next-line @elastic/eui/href-or-on-click */}
+            <EuiLink
+              data-test-subj="hostnameCellLink"
+              href={'?' + policyResponseUri.search}
+              onClick={(ev: React.MouseEvent) => {
+                ev.preventDefault();
```

Comment:
No - its different here. In this case, its not a navigate to app, but rather a direct router history update. @kevinlog is assigned to issue 230 which will (I assume) add an additional hook similar to the one you referenced above, but that will use react-router's `history.push`.

Reply:
@paul-tavares gotcha. 

re: using the same logic, I meant should `ev.preventDefault()` here be replaced with:
```
 try {
        if (onClick) {
          onClick(ev);
        }
      } catch (error) {
        ev.preventDefault();
        throw error;
      }

      if (ev.defaultPrevented) {
        return;
      }

      if (ev.button !== 0) {
        return;
      }

      if (
        ev.currentTarget instanceof HTMLAnchorElement &&
        ev.currentTarget.target !== '' &&
        ev.currentTarget.target !== '_self'
      ) {
        return;
      }

      if (ev.metaKey || ev.altKey || ev.ctrlKey || ev.shiftKey) {
        return;
      }

      ev.preventDefault();
```

Reply:
Yeah, I thought about that. I'm hoping we will have a true hook implemented soon so that we don't keep duplicating this type of code (increase tech. debt). There are a few links already doing this through out our codebase

@kevinlog you ok if I just go ahead and implement the hook for this?

Reply:
@paul-tavares yes, sounds good.

Reply:
Ok. If @oatkiller is ok with it, then I will work on that Issue next and will search/replace usages in our code base with the new callback.